kp_service_test/src/test/resources/templates/license/v3/read/testReadLicenseAfterRetire/response.json

```diff
@@ -1,14 +1,28 @@
 {
-  "id" : "api.license.read",
-  "ver" : "3.0",
-   "ts": "@matchesDatePattern('yyyy-MM-dd')@",
-  "params" : {
-    "resmsgid" : null,
-    "msgid" : null,
-    "err" : "RESOURCE_NOT_FOUND",
-    "status" : "failed",
-    "errmsg" : "License not found with identifier: ${identifier}"
-  },
-  "responseCode" : "RESOURCE_NOT_FOUND",
```

Comment:
Is it the functional requirement changes to provide Retired License data instead of throwing exception?

Reply:
This was the updated requirement I think. I need to confirm.